common/src/main/java/org/apache/sedona/Functions.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.sedona;
```

Comment:
Wasn't sure if it should go in any further package from here. Definitely open to suggestions on a lot of the naming involved.

Reply:
The package name should probably be org.apache.sedona.common.

It's safer to let each maven module have their own package namespace. For instance OSGi runtimes have a strict requirement on that.

Reply:
Fair point, will update